linera-chain/src/certificate/confirmed.rs

```diff
@@ -89,8 +81,21 @@ impl Serialize for GenericCertificate<ConfirmedBlock> {
     where
         S: serde::Serializer,
     {
-        let certificate: Certificate = self.clone().into();
-        certificate.serialize(serializer)
+        #[derive(Debug, Serialize)]
+        #[serde(rename = "ConfirmedBlockCertificate")]
+        struct Helper<'a> {
+            value: &'a ConfirmedBlock,
+            round: Round,
+            signatures: &'a Vec<(ValidatorName, Signature)>,
+        }
+
+        let helper = Helper {
+            value: self.inner(),
+            round: self.round,
+            signatures: self.signatures(),
+        };
+
+        helper.serialize(serializer)
```

Comment:
TODO: make consistent with `Serializer` impls on timeout and validatedblock certificates.